languages/elixir/exercises/concept/pattern-matching/.meta/design.md

```diff
@@ -0,0 +1,45 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to introduce the basics of pattern matching, focusing on matching within tuples and maps.
+
+## Learning objectives
+
+- Know what pattern matching is.
+- Know how to deconstruct a more complex data type into something simpler.
+
+## Out of scope
+
+- Head | Tail syntax in lists
+- Structs
+- Guard clauses
+- Matching on binaries
+- The pin operator
+
+## Prerequisites
+
+This exercise's prerequisites are:
+
+- `basics`: needs to be able to define named functions, return values
+
+## Concepts
+
+The concepts this exercise unlocks are:
+
+- `pattern-matching` know how to perform the most common types of pattern matches.
```

Comment:
```suggestion
- `pattern-matching-basic` know how to perform the most common types of pattern matches.
```

Reply:
@neenjaw That suggestion [goes against our naming rules for concepts](https://github.com/exercism/v3/blob/master/docs/maintainers/determining-concepts.md#naming-concepts). I think the original version is fine, although I would like for it to expand a bit on exactly what it will be unlocking.